### PR TITLE
feat: implement infra nginx (MAPCO-3316)

### PR DESCRIPTION
**Pull request by ronenkapelian**
# Overview
B2B services should be pass by common nginx service that connected to opa-la (auth)
# Related Issue / Discussion
PYCSW worked with additional nginx as part of deployment that contain opa + envoy for auth.
# Additional Information
The nginx + opa + envoy was removed and now pycsw service will be deployed with sub-chart of common nginx service and common opa.

### CODE REVIEW

**Review by asafMasa — APPROVED** [review]
👍

File: helm/raster/config/default.conf [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+server {
+    listen      {{ .Values.nginx.targetPort }};
+    # the domain name it will serve for
+    server_name pycsw;
+    # max upload size, adjust to taste
+    keepalive_timeout  500;
+    proxy_connect_timeout       600;
+    proxy_send_timeout          600;
```
Comment (EllaMartirosyan) [inline helm/raster/config/default.conf:21]: We need also:
proxy_read_timeout
And it should be configurable

File: helm/raster/config/default.conf [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    location /liveness {
+        return 200 "I'm alive\n";
+    }
+    location / {
+        set $original_method $request_method;
+        set $original_args $args;
+
+        {{ if .Values.nginx.authorization.enabled }}
+        set $domain {{ .Values.nginx.authorization.domain }};
+        auth_request /_validate_jwt;
```
Comment (EllaMartirosyan) [inline helm/raster/config/default.conf:38]: We need a solution for internal/external ports

File: helm/raster/config/nginx.conf [hunk 3]
```diff
@@ -1,41 +1,68 @@
...
+            '"connections_waiting": "$connections_waiting"'
+        '},'
+        #'"TraceId": "$opentelemetry_trace_id",' ## this is a byte sequence (hex-encoded in JSON)
+        #'"SpanId": "$opentelemetry_span_id",'
+        '"TokenUser": "$jwt_payload_sub",'
+        '"SeverityText": "INFO",'
+        '"SeverityNumber": 9,'
+        '"RequestBody": "$request_body",'
+        '"Body": "$request"'
+    '}';
```
Comment (EllaMartirosyan) [inline helm/raster/config/nginx.conf:51]: Log format should use MapColonies opentelemetry convention:
https://mapcolonies.atlassian.net/wiki/spaces/MAPConflicResolution/pages/1876328458/MapColonies+OpenTelemetry+Convention

**Pull request merged**